### PR TITLE
homework_2

### PR DESCRIPTION
**Pull request by batteryfree**

### CODE REVIEW

**Comment by markoffden** [comment]
well done, but pls fix do following adjustments

hw1: check user input to be a proper number (use parseInt), use ES6 syntax

hw2: think how to invalidate user input after each step, but not after both steps

**Comment by markoffden** [comment]
also you have linter issues, so
1. reinstall deps --- npm i
2. run tests --- npm run test
3. fix issues
4. upd homework
5. commit changes

good luck!

**Comment by batteryfree** [comment]
Corrected check

File: homeworks/roman.vlasov_batteryfree/homework_2/drawpyramid.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    const s1 = '#';
+    for (let i = 1; i <= hpyramid; i++) {
+        console.log(s.repeat(hpyramid - i) + s1.repeat(2 * i + 2));
+    }
+}
+const readline = require('readline');
+
+const readconsole = readline.createInterface(process.stdin, process.stdout);
+readconsole.question('Enter the height of the pyramid in the range from 1 to 20:', (answer) => {
+    const a = parseInt(answer, 2);
```
Comment (markoffden) [inline homeworks/roman.vlasov_batteryfree/homework_2/drawpyramid.js]: use 10 as radix argument

File: homeworks/roman.vlasov_batteryfree/homework_2/drawpyramid.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    if (a >= 1 && a <= 20) {
+        drawpyrapid(a);
+    } else {
+        console.log('***Invalid data entered***');
+    }
+    readconsole.close();
+});
```
Comment (markoffden) [inline homeworks/roman.vlasov_batteryfree/homework_2/drawpyramid.js]: doesn't work, tried with 12

File: homeworks/roman.vlasov_batteryfree/homework_2/palindromicnumbers.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+            if (typeof (num2) === 'number' && num1 < num2) {
+                palnum(num1, num2);
+            } else {
+                console.log('***Invalid data entered***');
+            }
+            readconsole.close();
+        });
+    } else {
+        console.log('***Invalid data entered***');
+        readconsole.close();
```
Comment (markoffden) [inline homeworks/roman.vlasov_batteryfree/homework_2/palindromicnumbers.js]: good work!

tip: try to follow DRY (Don't Repeat Yourself) principles as much as you can, e.g. readconsole.close() could be ran only once after you script. Pay attention to such details in your future tasks ;)

**Comment by batteryfree** [comment]
Fixed, check please

**Pull request merged**